code/modules/mob/living/silicon/pai/personality.dm

```diff
@@ -24,17 +24,17 @@
 
 // loads the savefile corresponding to the mob's ckey
 // if silent=true, report incompatible savefiles
-// returns 1 if loaded (or file was incompatible)
-// returns 0 if savefile did not exist
+// returns TRUE if loaded (or file was incompatible)
+// returns FALSE if savefile did not exist
 
 /datum/pai_candidate/proc/savefile_load(mob/user, silent = TRUE)
 	if (is_guest_key(user.key))
-		return 0
+		return FALSE
 
 	var/path = savefile_path(user)
 
 	if (!fexists(path))
-		return 0
+		return FALSE
 
 	var/savefile/F = new /savefile(path)
```

Comment:
better name can be done here

Reply:
I think savefiles are intentionally using single letter vars, even mothblock's new prefs has them.

Reply:
lmao

Reply:
cringe